python/tk_desktop/systray.py

```diff
@@ -163,14 +157,13 @@ def state(self, value):
             self._set_window_mask()
             self.__move_to_systray()
             self.setWindowFlags(self.windowFlags() | QtCore.Qt.FramelessWindowHint)
-            if osutils is not None:
-                osutils.make_app_background()
+            osutils.make_app_background()
 
         elif self.__state == self.STATE_WINDOWED:
             self._set_window_mask()
             self.setWindowFlags(self.windowFlags() & ~QtCore.Qt.FramelessWindowHint)
             self.show()
-            if osutils is None:
+            if osutils.is_mocked:
```

Comment:
Does this work if it is not mocked?
I don't see `osutils.is_mocked = False` being patched in, is it natively there?

Reply:
Also further down, (ln 326 -328), we are raising and then running `make_app_foreground`.
Is there a need to make this an if else? Can we just do both and remove the need for the `is_mocked`?

Reply:
Good point! I wasn't thinking enought about the code and just trying to replicate the flow.